packages/downloads/src/portal/portal-request-download-metadata.ts

```diff
@@ -67,30 +81,61 @@ export function portalRequestDownloadMetadata(
     });
 }
 
-function fetchLastEditDate(params: any): Promise<number | undefined> {
-  const { datasetId, url, type, modified, authentication } = params;
-
-  const layerId = datasetId.split("_")[1] || 0;
+function fetchCacheSearchMetadata(params: any): Promise<ICacheSearchMetadata> {
+  const { format, layerId, url, type, modified, authentication } = params;
 
   if (type !== "Feature Service" && type !== "Map Service") {
-    return Promise.resolve(modified);
+    return Promise.resolve({
+      modified,
+      format: getSearchFormat(format, false)
+    });
   }
 
-  return getLayer({
-    url: `${url}/${layerId}`,
-    authentication
-  }).then((layer: any) => {
-    const editingInfo: any | null = layer.editingInfo;
-    return editingInfo ? editingInfo.lastEditDate : undefined;
-  });
+  return getService({ url, authentication }).then(
+    (response: IFeatureServiceDefinition) => {
+      const layers: ILayerDefinition[] = response.layers;
+      const multilayer = isMultilayerRequest(layerId, layers);
+      return {
+        format: getSearchFormat(format, multilayer),
+        modified: extractLastEditDate(layers)
```

Comment:
Can layers be null or empty?

Reply:
ArcGIS rest-js indicate `layers` is non-optional, but I have now added handling in the event is comes back null or undefined.  Also added a test for that situation.